docs/plugin/custom-source.md

```diff
@@ -117,7 +117,9 @@ For a final implementation, see the [Botkube template repository](./quick-start.
    			case <-ctx.Done():
    				ticker.Stop()
    			case <-ticker.C:
-   				out.Output <- []byte("Ticker Event")
+   				out.Event <- source.Event{
```

Comment:
Please fix the indention here. Thanks 🙂 
![Screenshot 2023-03-08 at 15 49 05](https://user-images.githubusercontent.com/17568639/223744953-34ac4c10-1e37-4dfe-a984-cbdb2f21be66.png)

Reply:
![Screen Shot 2023-03-09 at 10 56 43](https://user-images.githubusercontent.com/1237982/223957249-b49fef80-a245-4387-b6d6-419b6ddabfea.png)